js/filteringBox.js

```diff
@@ -27,6 +27,78 @@ function openFilterModal() {
     closeFilterButton.classList.add("filter__box__topDiv__closeFilterBtn");
     closeFilterButton.innerHTML = "X";
     closeFilterButton.addEventListener("click", closeFilterModal);
+
+    // Checkbox divs
+    const checkDivMain = document.createElement("div");
+    filterBox.appendChild(checkDivMain);
+    checkDivMain.classList.add("filter__box__checkDivMain");
```

Comment:
The think I noticed, which are not very significant but still interesting.
.classList.add adds a class to an existing element without replacing the old ones.
While .className  assigns a new value to the element, but as I mentioned, it doesn't really matter here.